app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -89,19 +89,13 @@ def bulk_search
       param :search, String, :desc => N_('Resume tasks matching search string')
       param :task_ids, Array, :desc => N_('Resume specific tasks by ID')
       def bulk_resume
-        scope = resource_scope
-        scope = scope.search_for(params[:search]) if params[:search]
-        scope = scope.select('DISTINCT foreman_tasks_tasks.*')
         if params[:search].nil? && params[:task_ids].nil?
-          scope = scope.where(:state => :paused)
-          scope = scope.where(:result => :error)
+          raise ArgumentError, _('Please provide at least one of search or task_ids parameters in the request')
```

Comment:
See https://github.com/theforeman/foreman-tasks/pull/517/files#r394391514

Reply:
Changed to `BadRequest`, Thanks!